Sources/MapboxCoreNavigation/CacheHandlerFactory.swift

```diff
@@ -0,0 +1,25 @@
+
+import MapboxNavigationNative
+
+struct CacheHandlerFactory {
```

Comment:
Using enum will make it clear that this class isn't supposed to be created (swiftc won't allow that).

Reply:
Good catch, done